frontend/app-preview/src/views/LandingPage.module.css

```
@@ -21,4 +21,6 @@
   display: flex;
   flex-direction: column;
   height: calc(100vh - var(--header-height));
+  gap: var(--fds-spacing-8);
```

Use the same spacing as above the togglebuttons and dropdpwn menu 😁

The best solution would probably be to create a variable called `--preview-spacing` or something that is set to `var(--fds-spacing-4)` and use that variable for both this gap and for paddig-top in PreviewControlHeader. But no biggie

```suggestion
  gap: var(--fds-spacing-4);
```